lib/ui/mockData/MockLandingPageViewModel.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:farmsmart_flutter/ui/LandingPage.dart';
+import 'package:intl/intl.dart';
+
+class _Strings {
+  static final String detailText = Intl.message("Kuwezesha wakenya elimu inayohitajika kuwa wakulima wenye mafanikio.");
```

Comment:
these are not final. if we change the language this string will not change!
Instead in the widget do Intl.message(_Strings.detailText) and in the _String class just have the english string.